frontend/components/account/js/account-billing.component.ts

```diff
@@ -78,29 +79,47 @@ class AccountBillingController implements ng.IController {
 		return this.APIService.get(this.account + "/invoices")
 			.then((response) => {
 				this.billings = response.data;
+			})
+			.catch((error) => {
+				console.error(error);
 			});
 	}
 
 	public initPlans() {
-		return this.APIService.get("plans")
+		return this.APIService.get("/plans")
 			.then((response) => {
 				if (response.status === 200) {
```

Comment:
Is this check necessary (I know you probably didn't write this)? according to angular docs:
> A response status code between 200 and 299 is considered a success status and will result in the success callback being called. Any response status code outside of that range is considered an error status and will result in the error callback being called. Also, status codes less than -1 are normalized to zero. -1 usually means the request was aborted, e.g. using a config.timeout. More information about the status might be available in the xhrStatus property.

So we can assume that if the promise resolves it's a success? The other API calls on this page don't check response status numbers,